packages/rebox/android/android/gradle.properties

```diff
@@ -23,3 +23,6 @@ org.gradle.parallel=true
 android.useAndroidX=true
 # Automatically convert third-party libraries to use AndroidX
 android.enableJetifier=true
+
+# Version of flipper SDK to use with React Native
+FLIPPER_VERSION=0.99.0
```

Comment:
I added this, because React Native upgrade helper showed it should be updated, but I'm not sure if it's needed here?